dashboard/app/controllers/levels_controller.rb

```diff
@@ -35,10 +35,6 @@ def edit
     if @level.is_a? Grid
       @level.maze_data = @level.class.unparse_maze(@level.properties)
     end
-    if @level.is_a? DSLDefined
-      @filename = @level.filename
-      @dsl_file = File.read(@filename) if @filename && File.exist?(@filename)
-    end
```

Comment:
Just to make sure I followed right, this logic was refactored to reuse [dsl_text](https://github.com/code-dot-org/code-dot-org/pull/3397/files#diff-1514f30aa6e8764312feafd5e50f793fR26) instead?